rootfs_linux.go

```diff
@@ -62,6 +69,17 @@ func setupRootfs(config *configs.Config, console *linuxConsole) (err error) {
 	return nil
 }
 
+func mountCmd(cmd string) error {
+	if cmd != "" {
+		args := strings.Split(cmd, " ")
+		out, err := exec.Command(args[0], args[1:]...).CombinedOutput()
+		if err != nil {
+			return fmt.Errorf("%s failed: %s", cmd, string(out))
```

Comment:
Maybe also return err embedded inside so we can catch issues with exec'ing the command itself?